### PR TITLE
Fix strings highlighting

### PR DESCRIPTION
**Pull request by ccfontes**

### CODE REVIEW

File: shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml [hunk 1]
```diff
@@ -44,7 +44,10 @@ contexts:
     - match: '[a-z_][a-zA-Z0-9_.-]*'
       scope: variable.other.readwrite.shards
   strings:
-    - match: '".*"'
-      scope: string.quoted.double.shards
-    - match: '""".*"""'
```
Comment (sinkingsugar) [inline shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml:49]: Don't we need those?

File: shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml [hunk 2]
```diff
@@ -44,7 +44,10 @@ contexts:
...
-      scope: string.quoted.triple.shards
+    - match: '"|"""'
+      scope: string.quoted.shards
+      push:
+        - meta_content_scope: string.quoted.shards
+        - match: '"|"""'
```
Comment (sinkingsugar) [inline shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml:51]: why the `|`?

File: shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml (hunk 1 above)
Comment (ccfontes) [reply shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml:49]: No, that's what's causing the issue.

File: shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml (hunk 2 above)
Comment (ccfontes) [reply shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml:51]: To match a double quote or a triple double quote.
I'm going to make it a bit more correct.

Comment (sinkingsugar) [reply shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml]: Oh so unrelated to our `|`?

Comment (ccfontes) [reply shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml]: Yup, unrelated, it's just an "or" for a regex

**Comment by ccfontes** [comment]
Fixed by #725
CC #Doc-105

**Pull request closed without merging**